Project.toml

```diff
@@ -12,7 +12,7 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-MathOptInterface = "1.3.0"
+MathOptInterface = "1.7"
```

Comment:
@blegat we need to remember to update this when we rely on new MOI features

Reply:
Indeed, good catch